specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json

```diff
@@ -1993,6 +1993,266 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/workflows/{workflowName}/runs/{runName}/actions/{actionName}/repetitions/{repetitionName}/requestHistories": {
+      "post": {
```

Comment:
Please confirm it's expected, since your operation is called "GET". IOW, please confirm not a typo.

Reply:
This is a typo. Fixed